docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -291,46 +291,69 @@ The REST API source will try to automatically handle pagination for you. This wo
-| [OffsetPaginator](../../general-usage/http/rest-client.md#offsetpaginator) | `offset` | The pagination is based on an offset parameter. With total items count either in the response body or explicitly provided. |
-| [PageNumberPaginator](../../general-usage/http/rest-client.md#pagenumberpaginator) | `page_number` | The pagination is based on a page number parameter. With total pages count either in the response body or explicitly provided. |
-| [JSONCursorPaginator](../../general-usage/http/rest-client.md#jsonresponsecursorpaginator) | `cursor` | The pagination is based on a cursor parameter. The value of the cursor is in the response body (JSON). |
-| SinglePagePaginator | `single_page` | The response will be interpreted as a single-page response, ignoring possible pagination metadata. |
-| `None` | `auto` | Explicitly specify that the source should automatically detect the pagination method. |
+```json
+{
+    "data": [
+        {"id": 1, "title": "Post 1"},
+        {"id": 2, "title": "Post 2"},
+        {"id": 3, "title": "Post 3"}
+    ],
+    "pagination": {
+        "next": "https://api.example.com/posts?page=2"
+    }
+}
+```
 
-To specify the pagination configuration, use the `paginator` field in the [client](#client) or [endpoint](#endpoint-configuration) configurations. You may either use a dictionary with a string alias in the `type` field along with the required parameters, or use the paginator instance directly:
+You can configure the pagination for the `posts` resource like this:
 
 ```py
 {
-    # ...
+    "path": "posts",
     "paginator": {
-        "type": "json_links",
-        "next_url_path": "paging.next",
+        "type": "json_response",
+        "next_url_path": "pagination.next",
     }
 }
 ```
 
-Or using the paginator instance:
+Alternatively, you can use the paginator instance directly:
 
 ```py
+from dlt.sources.helpers.rest_client.paginators import JSONResponsePaginator
+
+# ...
+
 {
-    # ...
+    "path": "posts",
     "paginator": JSONResponsePaginator(
-        next_url_path="paging.next"
+        next_url_path="pagination.next"
     ),
 }
 ```
 
-This is useful when you're [implementing and using a custom paginator](../../general-usage/http/rest-client.md#custom-paginator).
+:::note
+Currently pagination is supported only for GET requests. To handle POST requests with pagination, you need to implement a [custom paginator](../../general-usage/http/rest-client.md#custom-paginator).
+:::
+
+These are the available paginators:
+
+| `type` | Paginator class | Description |
+| ------------ | -------------- | ----------- |
+| `json_response` | [JSONResponsePaginator](../../general-usage/http/rest-client.md#jsonresponsepaginator) | The link to the next page is in the body (JSON) of the response.<br/>*Parameters:*<ul><li>`next_url_path` (str) - the JSONPath to the next page URL</li></ul> |
```

Comment:
The bold may look better here then cursive

```suggestion
| `json_response` | [JSONResponsePaginator](../../general-usage/http/rest-client.md#jsonresponsepaginator) | The link to the next page is in the body (JSON) of the response.<br/>**Parameters:**<ul><li>`next_url_path` (str) - the JSONPath to the next page URL</li></ul> |
```

Reply:
@VioletM yes, I'll make it a separate sections as in RESTClient, and use bold font for those.